examples/autograd/singa-onnx/onnx_model.py

```diff
@@ -0,0 +1,43 @@
+import pickle
+from singa import tensor
+from singa.tensor import Tensor
+from singa import autograd
+from singa import optimizer
+import numpy as np
+import onnx
+from onnx import numpy_helper
+
+def load_onnx_model(name = 'singonnx.pkl'):
```

Comment:
should the two functions be put into autograd.py, i.e., included in singa py code?

Reply:
yes, I moved the load_onnx_model to sonnx.py(short for singa-onnx)
the load_onnx_model function will not confuse with autograd functions